zebra/rt_netlink.c

```diff
@@ -1264,6 +1386,67 @@ static bool _netlink_route_build_singlepath(const struct prefix *p,
 					      sizeof(label_buf)))
 		return false;
 
+	if (nexthop->nh_seg6local_ctx) {
+		struct rtattr *nest;
+		const struct seg6local_context *ctx;
+
+		ctx = nexthop->nh_seg6local_ctx;
+		nl_attr_put16(nlmsg, req_size, RTA_ENCAP_TYPE,
```

Comment:
None of this encode code checks the return from the nl_attr_xxx apis?

Reply:
It's needed. I didn't care that. need fix.
i'm gonna be writing it right now.

Thank you. please wait soon.

Reply:
I fixed it.
All nl_attr_{nest, put} are with error check. (exclude existing lines before this PR)

https://github.com/FRRouting/frr/pull/5865/commits/550b2b82f735cae7026c0c2f246e543713cac129